argoverse/data_loading/simple_track_dataloader.py

```diff
@@ -124,7 +124,7 @@ def get_ordered_log_ply_fpaths(self, log_id: str) -> List[str]:
         Args:
             log_id: str, unique ID of vehicle log
         Returns:
-            ply_fpaths: List of strings, representing paths to ply files in this log
+            ply_fpaths: List of strings, representing paths to ordered ply files in this log
```

Comment:
Potentially mention how `sorted` determines ordering?

Reply:
good point, just added it